src/clipper/argparser.py

```diff
@@ -728,9 +728,9 @@ def getArgParser() -> argparse.ArgumentParser:
     )
 
     ytdl_options.add_argument(
-        "--cookiefile",
-        "-cf",
-        dest="cookiefile",
+        "--cookies",
+        "-c",
+        dest="cookies",
```

Comment:
Can you change the metavar to `FILE`. If we change `dest` to `cookiefile` then it might be clearer in the code as well that we expect this to be a file path. This would be more consistent with yt-dlp's code as well:
```
    filesystem.add_option(
        '--cookies',
        dest='cookiefile', metavar='FILE',
        help='Netscape formatted file to read cookies from and dump cookie jar in')
```

Reply:
I think changing the option from `--cookiefile` to `--cookies` is less clear and is an unnecessary change to yt_clipper's CLI. We just need to use the right option name for yt-dlp.